kopf/reactor/peering.py

```diff
@@ -231,12 +229,16 @@ def apply_peers(
         if not freeze.is_set():
             logger.info(f"Freezing operations in favour of {prio_peers}.")
             freeze.set()
+    
     else:
         if same_peers:
             logger.warning(f"Possibly conflicting operators with the same priority: {same_peers}.")
-        if freeze.is_set():
-            logger.info(f"Resuming operations after the freeze.")
-            freeze.clear()
+            logger.warning(f"Freezed all Operators: {peers}")
+            freeze.set()
+        else:
+            if freeze.is_set():
+                logger.info(f"Resuming operations after the freeze. Conflicting operators with the same priority are gone")
+                freeze.clear()
```

Comment:
Can this be done via one-level `if` (just to make it more readable; the logic is fine)?

```python
if prio_peers:
    ...
elif same_peers:
    ...
elif freeze.is_set():
    ...
```

Previously, there was a 2-step block (if same_peers + if freeze.is_set()). Now, it is either-or, so the nested if's are not needed.

Reply:
Sure ! I agree. Code Updated !